parsec-3.0/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM debian:latest
+MAINTAINER Ouamar Sais
+
+#Set the current directory to /tmp
+WORKDIR /tmp 
+
+
+#Install needed libraries for Parsec build
+RUN apt-get update && apt-get install -y \
+	wget\
+	xorg-dev\
+	make\
+	patch\
+	g++ \
+	m4\
+	gettext\
+	libtbb-dev
+	
+
+#Download, untar, and delete Parsec archive
+RUN wget -nv http://parsec.cs.princeton.edu/download/3.0/parsec-3.0.tar.gz && \
+	tar -xvf /tmp/parsec-3.0.tar.gz && \
+	mv /tmp/parsec-3.0 /home/ && \
+	rm /tmp/parsec-3.0.tar.gz && \
+	find /home/parsec-3.0/ -name \*.tar -exec xz -9 \{\} \;
+
+#Put the patch nedded for ssl build int its src directory
+COPY pod.patch /home/parsec-3.0/
+
+#Copy scripts for getting build configurations, building parsec and running parsec, as well as help text file  
+COPY getCompilConf.sh build.sh run.sh help.txt htu.txt /home/parsec-3.0/
+
+#Apply patch to fix ssl build
+RUN cd /home/parsec-3.0/pkgs/libs/ssl/src/ && \
+	patch -p1 < /home/parsec-3.0/pod.patch
+
+
+#Set current directory to /home/parsec-3.0
+WORKDIR /home/parsec-3.0
+
+#Launch Parsec compilation
+RUN sh /home/parsec-3.0/getCompilConf.sh && \
```

Comment:
Move all RUN command for Parsec in one command to get a smaller image, and then don't forget to clean and to remove all the sources